# HG changeset patch
# User "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
# Date 1322261875 -3600
# Node ID e6b5bd48194274dc184c37091d6ff2b3c1ff305e
# Parent  b6faa28e76e067be1306019b64120492f2fc655d
config/target: add float support selection

Changeset #149c33923f47 broke the architectures that do not
support the --with-float=X ./configure flag (in gcc). For example,
x86_64 does not support it.

Add a new blind config option that architectures can set to tell
they support floating point selection.

Reported-by: Morten Thunberg Svendsen <mts@doredevelopment.dk>
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>

diff --git a/config/target.in b/config/target.in
--- a/config/target.in
+++ b/config/target.in
@@ -14,6 +14,7 @@
 config ARCH_SUPPORTS_WITH_ABI
 config ARCH_SUPPORTS_WITH_CPU
 config ARCH_SUPPORTS_WITH_TUNE
+config ARCH_SUPPORTS_WITH_FLOAT
 config ARCH_SUPPORTS_WITH_FPU
 config ARCH_SUPPORTS_SOFTFP
 
@@ -148,6 +149,9 @@
 config ARCH_SUPPORTS_WITH_TUNE
     bool
 
+config ARCH_SUPPORTS_WITH_FLOAT
+    bool
+
 config ARCH_SUPPORTS_WITH_FPU
     bool
 
